src/main/webapp/components/FilterList.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+  <div>
+    <!-- User can set their own filters. -->
+    <v-combobox v-model="filters"
+                label="Active Filters"
+                hint="Add your own"
+                multiple chips deletable-chips>
+    </v-combobox>
+    <!-- Or select from common filters -->
+    <v-checkbox v-model="filters"
+                v-for="filter in suggestedFilters" 
+                :key="filter"
+                :label="filter" 
+                :value="filter">
+    </v-checkbox>
+  </div>
+</template>
+
+<script>
+module.exports = {
+  props: {
+    value: Array,
+    school: String, // TODO: Modify SearchBar.vue to emit school and course to NoteGrid, and pass to FilterList
+    course: String,
+  },
+  data: function() {
+    return {
+      suggestedFilters: [
+        "MAC",
+        "ENC",
+        "COP",
+      ]
+    }
+  },
+  computed: {
+    filters: {
+      get: function() {
+        return this.value;
+      },
+      set: function(filters) {
+        this.$emit('input', filters)
+      }
+    },
+  },
+  methods: {
+    fetchSuggestedLabels: function() {
+      let url = new URL("/common-labels", window.location.href);
```

Comment:
Is this the servlet for retrieving all the school and course labels? Or is this something else?

Reply:
If it is a call to the servlet for retrieving all the school and course labels, I named the path `/get-school-and-course-labels` but we could also rename it on #57 if needed.

Reply:
Oops, forgot a TODO. This is a servlet that does not exist yet haha. It should serve some of the most common labels for each school/course.
```suggestion
      let url = new URL("/common-labels", window.location.href); // TODO: Create the servlet and rename "/common-labels"
```

Reply:
ohhhh gotcha, then ignore my comment on aligning with the servlet in #57!

Reply:
I can go ahead and create the `/common-labels` servlet today, I'll add it to the project board.

Reply:
Now that I think about it though, I'm not sure we'd be able to create a `/common-labels` servlet without making a modification to the schema to track the number of times any given label has been searched for. Like each label is only 1 row in the Labels table at this point so I think to get the 10 most common for example, we'd need to add a column on the Labels table `numTimesSearched` or something that is basically just a counter that goes up by 1 everytime that label is searched for. Then we could simply query for labels ordered descending by `numTimesSearched`.
I could definitely work on this but I'm not sure if this is something that we want to include in our scope for this week or whether this should be considered a moonshot. If it isn't something we want to get done immediately, then it'd be good to make changes here to reflect that and assume we don't have a `/common-labels` path.

Reply:
I was thinking the most common labels assigned to the notes on the DB, not searched for. So something like
```
SELECT count(label) FROM
  miscLabels INNER JOIN (
    SELECT noteID FROM notes
    WHERE course="course" AND school="school"
  ) 
GROUP BY label
SORT BY count(label) DESC
 ```
I'm sure this will require a lot of tweaking, but no need to change the schema, I think.